Documentation/network/kubernetes/kubeproxy-free.rst

```diff
@@ -324,6 +324,35 @@ be lost on its path to the service endpoint.
   or :ref:`Hybrid<Hybrid Mode>` mode if only TCP-based services are exposed to the outside
   world for the latter.
 
+Internal Traffic Policy
+***********************
+
+Similar to ``externalTrafficPolicy`` described above, Cilium's eBPF kube-proxy replacement
+supports ``internalTrafficPolicy``, which translates the above semantics to in-cluster traffic.
+
+- For services with ``internalTrafficPolicy=Local``, traffic originated from pods in the
+  current cluster is routed only to endpoints within the same node the traffic originated from.
+
+- ``internalTrafficPolicy=Cluster`` is the default, and it doesn't restrict the endpoints that
+  can handle internal (in-cluster) traffic.
+
+The following table gives an idea of what backends are used to serve connections to a service,
+depending on the external and internal traffic policies:
+
++---------------------+-------------------------------------------------+
+| Traffic policy      | Service backends used                           |
++----------+----------+-------------------------+-----------------------+
+| Internal | External | for North-South traffic | for East-West traffic |
++==========+==========+=========================+=======================+
+| Cluster  | Cluster  | All                     | All                   |
```

Comment:
just a tiny nit: could you change `All` into `All (default)` so it is obvious to readers what is the default behavior?